### ISSUE TITLE
[firewalld 0.7.2] testsuite: 44 failed

### ISSUE THREAD

**Issue by mbiebl** [firewalld/firewalld#528]
Test "44: ICMP block" fails reliably on Debian sid.
The test suite is run via autopkgtest inside qemu.
I've run the test-suite 10 times in a row and test 44 failed reliably each time.
Full log attached.
[log.txt](https://github.com/firewalld/firewalld/files/3711789/log.txt)

**Comment by mbiebl** [comment]
I can attach the logs from the other runs as well, if needed

**Comment by erig0** [comment]
I think this is a bug in the nftables CLI frontend. I think it went away in master when we converted to libnftables JSON. So we'll have to find a fix for the stable branches.

**Comment by erig0** [comment]
This is been confirmed as an nftables bug. You can recreate with
```
# nft --echo --handle add rule inet t c meta l4proto '{icmp, icmpv6} accept'
W: Received event for an unknown set.W: Received event for an unknown set.add rule inet t c accept # handle 7
netlink: Error: Unknown set '__set1' in lookup expression
```

One of my colleagues is taking a look at it.

**Comment by erig0** [comment]
nftables upstream fix: `8ecb88558959 ("src: restore --echo with anonymous sets")`
